### PR TITLE
chore: fix symbol error

### PR DESCRIPTION
**Pull request by tiaoxizhan**

### CODE REVIEW

File: api/rs/slint/android.rs [hunk 1]
```diff
@@ -33,7 +33,7 @@
 //! }
 //! ```
 //!
-//! That function must be in a `cdylib` library, and you should enable the "backend-android-activity-06"`
+//! That function must be in a `cdylib` library, and you should enable the `backend-android-activity-06"`
```
Comment (hunger) [inline api/rs/slint/android.rs:36]: I think the trailing `\`` is the issue here, not the starting `"`.

File: api/cpp/esp-idf/slint/README.md [hunk 2]
```diff
@@ -189,7 +189,7 @@ If your display controller expects little endian, set the `color_swap_16` field
 You see errors at application link time such as these:
 
 ```
-compiler_builtins.4c2482f45199cb1e-cgu.05:(.text.__udivdi3+0x0): multiple definition of `__udivdi3'; .../libgcc.a(_udivdi3.o): first defined here
+compiler_builtins.4c2482f45199cb1e-cgu.05:(.text.__udivdi3+0x0): multiple definition of `__udivdi3`; .../libgcc.a(_udivdi3.o): first defined here
```
Comment (hunger) [inline api/cpp/esp-idf/slint/README.md:192]: This was cut and pasted, I think there is no need to fix that.

**Review by hunger — CHANGES_REQUESTED** [review]

File: api/rs/slint/android.rs (hunk 1 above)
Comment (tiaoxizhan) [reply api/rs/slint/android.rs]: Yes, I made a mistake..

Modified. Please review again.

File: api/cpp/esp-idf/slint/README.md (hunk 2 above)
Comment (tiaoxizhan) [reply api/cpp/esp-idf/slint/README.md]: Reverted

**Comment by ogoffart** [comment]
Thanks!

**Pull request merged**